project1/styles.css

```diff
@@ -0,0 +1,60 @@
+h1 {
+    font-size: 70px;
+    text-align: center;
+    background-color: darkgrey;
+    font-family: 'IM Fell French Canon SC', serif;
+    color: #034247;
+
+}
+
+h2 {
+    font-size: 20px;
+    text-align: center;
+    background-color: darkgrey;
```

Comment:
This matches the `background-color` of the `body` element, so you don't need to repeat it here.
```suggestion
```